application_configuration.md

```diff
@@ -106,12 +90,15 @@ gem 'devise-i18n'
 gem 'figaro'
 gem 'font-awesome-rails'
 gem 'goldiloader'
+gem 'http_accept_language'
+gem 'i18n-docs'
 gem 'jquery-rails'
 gem 'jquery-turbolinks'
 gem 'pg'
 gem 'rack-google-analytics'
 gem 'rails'
 gem 'rails-i18n'
+gem 'rails_real_favicon'
```

Comment:
It it really worth the dependencies (`rails < 5`, `rest-client ~> 1.8`), the few *github* stars, the extra config file and the extra [config line](https://github.com/RealFaviconGenerator/rails_real_favicon/blob/master/test/dummy/config/application.rb#L7)?

If it was, I would at least put it to the *development* environment section.